tests/Sdk/Unit/Extension/ValueResolver/StaticTextValueResolverTest.php

```diff
@@ -1,98 +0,0 @@
-<?php
```

Comment:
Please keep it as long as testable class exists. StaticTextValueResolver must be marked as deprecated.